src/configeditor/EditBoundingBoxes.tsx

```diff
@@ -0,0 +1,208 @@
+  const { type, bounds, values, setValues, error } = props;
+
+  const onChange = (value: string, field: "westmost" | "southmost" | "eastmost" | "northmost") => {
+    if (!bounds) {
+      const newBounds = (
+        field === "westmost" ? new BoundingBox(value, "", "", "") :
+        field === "southmost" ? new BoundingBox("", value, "", "") :
+        field === "eastmost" ? new BoundingBox("", "", value, "") :
+        new BoundingBox("", "", "", value)
+      );
+      setValues({
+        ...values,
+        boundingBoxes: {
+          ...values.boundingBoxes,
+          [type]: newBounds.toConfigBbox()
+        }
+      });
+    } else {
+      bounds[field] = value;
+      setValues({
+        ...values,
+        boundingBoxes: {
+          ...values.boundingBoxes,
+          [type]: bounds.toConfigBbox()
+        }
+      });
+    };
+  };
+
+  return (
+    <HStack align="start">
+      <LatLngInput
+        title="West"
+        field="westmost"
+        value={bounds ? bounds.westmost : ""}
+        onChange={e => onChange(e.target.value, "westmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="South"
+        field="southmost"
+        value={bounds ? bounds.southmost : ""}
+        onChange={e => onChange(e.target.value, "southmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="East"
+        field="eastmost"
+        value={bounds ? bounds.eastmost : ""}
+        onChange={e => onChange(e.target.value, "eastmost")}
+        error={error}
+      />
+      <LatLngInput
+        title="North"
+        field="northmost"
+        value={bounds ? bounds.northmost : ""}
+        onChange={e => onChange(e.target.value, "northmost")}
+        error={error}
+      />
+      <button
+        key={type}
+        title={"Clear input"}
+        onClick={() => setValues({
+          ...values,
+          boundingBoxes: {
+            ...values.boundingBoxes,
+            [type]: null
+          }
+        })}
+      >
+        &#x2715;
+      </button>
+    </HStack>
+  )
+}
+
+function EditBoundingBoxes() {
+  const { status, values, setValues, errors, submit } = useConfigEdit(["boundingBoxes"]);
+
+  // bounding boxes of different types of map
+  const boundingBoxes = values.boundingBoxes as Config["boundingBoxes"];
+  const defaultBounds = boundingBoxes.default ? new BoundingBox(...boundingBoxes.default) : null;
+  const warningAreaBounds = boundingBoxes.warningAreas ? new BoundingBox(...boundingBoxes.warningAreas) : null;
+  const rainMapBounds = boundingBoxes.rainMap ? new BoundingBox(...boundingBoxes.rainMap) : null;
+  const floodModelMapBounds = boundingBoxes.floodModelMap ? new BoundingBox(...boundingBoxes.floodModelMap) : null;
+  const damBounds = boundingBoxes.dams ? new BoundingBox(...boundingBoxes.dams) : null;
+
+  const boundingBoxErrors = "boundingBoxes" in errors ? errors.boundingBoxes as ErrorObject : {};
+
+  return (
+    <VStack align="left">
```

Comment:
I feel the page needs one or two sentences of explanation -- that the values should be WGS84 coordinates, and the 'default' is required and will be used if any of the others are missing.

Probably also need to explain briefly what each of the others is used for, in small letters under their heading, but that can be added later.

Reply:
Added some explanation on top of the page:

> Notes:
> - Values of bounding boxes should be in WGS84 coordinate system.
> - Default field is required and will be used in place of any missing field.